SPM Sample/SPM-sample-app.xcodeproj/xcshareddata/xcschemes/SPM-sample-app.xcscheme

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why is this file added now? 🤔

Reply:
I don't know, maybe added by my xcode 15